### PR TITLE
Forces optimize_memory_usage to False and add custom_objects argument in record_video.py

### PR DESCRIPTION
**Pull request by Affonso-Gui**
## Description
Fixes https://github.com/DLR-RM/rl-baselines3-zoo/issues/279.

## Motivation and Context
- [x] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
Haven't really took the time to add tests for this (and/or maybe for the same `enjoy.py` features). Let me know if I should.

### CODE REVIEW

**Comment by araffin** [comment]
Please update the changelog and we are good to go ;)

**Review by araffin — DISMISSED** [review]
LGTM, thanks =)

**Comment by Affonso-Gui** [comment]
> Please update the changelog

Done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 ### New Features
 
 ### Bug fixes
+- Added `custom_object` parameter on record_video.py
```
Comment (araffin) [inline CHANGELOG.md:8]:
```suggestion
- Added `custom_object` parameter on record_video.py (@Affonso-Gui) 
```

**Review by araffin — APPROVED** [review]

**Pull request merged**